pkg/cli/create.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kubebuilder/pkg/cli/internal"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+)
+
+func (c cli) newCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Scaffold a Kubernetes API or webhook",
+		Long:  `Scaffold a Kubernetes API or webhook.`,
+	}
+	cmd.AddCommand(c.newCreateAPICmd())
+
+	if !internal.ConfiguredAndV1() {
```

Comment:
```suggestion
	if !internal.IsConfigured() && !internal.IsV1()  {
```

Reply:
IHMO it is more clear and does not hurt the cohesion.

Reply:
Is the logic here correct? 
Should not we check if isConfigured and if is not V1 to do both scaffolds?

Reply:
`kubebuilder create webhook` needs to be added for any version except for v1.  `NOT (A & B) != (NOT A) & (NOT B)` so the logic was correct but your suggestion isn't. Your suggestion will only add the subcommand for not configured projects, and you can't use `kubebuilder create webhook` in a not configured projects, as it requires a `kubebuilder init` first.